tests/fixtures/music_nerd_pro_llm_anthropic/combination_responses_with_history_direct.hocon

```diff
@@ -39,7 +39,7 @@
             "text": "Where were they from?",
             "response": {
                 "text": {
-                    "keywords": ["Liverpool", "\"running_cost"\": 6.0]
+                    "keywords": ["Liverpool", "6.0"]
```

Comment:
Hmm. In retrospect, this was invalid JSON to begin with. HOCON can be a bit more lenient.